Doctrine/Subscriber/AbstractEntitySubscriber.php

```diff
@@ -207,7 +207,7 @@ protected function flushUpserts(): void
                     // If the entity isn't able to be sent to Salesforce for any reason,
                     // a RuntimeException is thrown. We don't want that stopping our fun.
                     $this->logger->error($e->getMessage());
-                    $this->logger->debug($e->getTraceAsString());
+                    $this->logger->debug('AbstractEntitySubscriber-001 - '.$e->getTraceAsString());
```

Comment:
Can you change this to "Exception occurred while flushing upserts " or "Exception occured in flushUpserts" or something along those lines?